src/screenControl.py

```diff
@@ -105,9 +106,9 @@ def outputSide(self):
         if self.mode == COMMAND_MODE:
             usageLines = processInput.USAGE_COMMAND.split('\n')
         for index, usageLine in enumerate(usageLines):
-            self.stdscr.addstr(self.getMinY() + index, borderX + 2, usageLine)
+            self.stdscr.addstr(self.getMinY() + index, borderX + 2, usageLine, curses.color_pair(0))
```

Comment:
Yeah, let's do that. Actually we can later make our chrome colored which would be sweet. For later.